Parse/ParseInstallation.cs

```diff
@@ -220,13 +220,13 @@ public ParseInstallation()
     internal override Task SaveAsync(Task toAwait, CancellationToken cancellationToken) {
       Task platformHookTask = null;
       if (CurrentInstallationController.IsCurrent(this)) {
-        SetIfDifferent("deviceType", ParseClient.PlatformHooks.DeviceType);
-        SetIfDifferent("timeZone", ParseClient.PlatformHooks.DeviceTimeZone);
-        SetIfDifferent("localeIdentifier", GetLocaleIdentifier());
-        SetIfDifferent("parseVersion", GetParseVersion().ToString());
-        SetIfDifferent("appVersion", ParseClient.PlatformHooks.AppBuildVersion);
-        SetIfDifferent("appIdentifier", ParseClient.PlatformHooks.AppIdentifier);
-        SetIfDifferent("appName", ParseClient.PlatformHooks.AppName);
+        ForceSetIfDifferent("deviceType", ParseClient.PlatformHooks.DeviceType);
```

Comment:
`SetIfDifferent` will not check for mutability. https://github.com/ParsePlatform/Parse-SDK-dotNET/blob/master/Parse/ParseObject.cs#L1238

I'm not sure this is the right fix.